frontend/src/components/Common/Chart/index.tsx

```diff
@@ -27,15 +27,19 @@ interface ChartData {
   }[];
 }
 
-function downloadCsv(chartData: ChartData, filename: string) {
+function downloadCsv(
```

Comment:
This is a bit counter-intuitive to me. It doesn't feel like this should be the responsibility of "downloadCSV". Whoever is calling downloadCSV should prepare the data.

Also, it looks like we have two functions `downloadCsv` that do basically the same thing, so we need to homogenize this.

I would actually suggest inverting the behavior, keep the full date in the charts data, but have an option to display time only when we visualize the chart. What do you think?